### PR TITLE
TOS reorganization

### PR DESCRIPTION
**Pull request by kdorheim**
Update the organization of the tos processing data that match the new structure of the data.

### CODE REVIEW

File: scripts/A5b.tos_HL.py [hunk 1]
```diff
@@ -147,14 +156,15 @@ def mean_HL_tos(path):
 catalog = catalog.search(require_all_on=["source_id"], **query)
 catalog = catalog.df.copy().reset_index(drop=True)
 catalog = catalog.loc[catalog['member_id'].str.contains('p1')].copy().reset_index(drop=True)
+catalog.to_csv("catalog.csv")
 
 # Set up the output directory and process the files.
-outdir = './HL/'
+outdir = '../tos/HL/'
```
Comment (leeyap) [inline scripts/A5b.tos_HL.py]: One or two periods here?

File: scripts/A5c.tos_LL.py [hunk 2]
```diff
@@ -148,9 +158,10 @@ def mean_LL_tos(path):
 catalog = catalog.search(require_all_on=["source_id"], **query)
 catalog = catalog.df.copy().reset_index(drop=True)
 catalog = catalog.loc[catalog['member_id'].str.contains('p1')].copy().reset_index(drop=True)
+catalog.to_csv("catalog_LL.csv")
 
 # Set up the output directory and process the files.
-outdir = './LL/'
+outdir = '../tos/LL/'
```
Comment (leeyap) [inline scripts/A5c.tos_LL.py]: same as above

File: scripts/B5b.process_historical_regional_tos.R [hunk 3]
```diff
@@ -0,0 +1,170 @@
+# After running the pangeo code scripts 0.global_tos.py, 0.HL_tos.py, and 0.LL_tos.py to get
```
Comment (leeyap) [inline scripts/B5b.process_historical_regional_tos.R]: Modify header for consistency?

**Review by leeyap — CHANGES_REQUESTED** [review]
Looks good, @kdorheim - thanks! Few minor comments below.

File: scripts/A5b.tos_HL.py (hunk 1 above)
Comment (kdorheim) [reply scripts/A5b.tos_HL.py]: gah yea I did a poor practice of setting up relative paths (😬 ) the latest commit should fix that 👍

**Review by leeyap — APPROVED** [review]
Looks good, thank you! Okay to merge?

**Pull request merged**